src/common/servlist.c

```diff
@@ -193,6 +193,11 @@ static const struct defaultserver def[] =
 #endif
 	{0,			"irc.globalgamers.net"},
 
+#ifdef USE_OPENSSL
```

Comment:
Unless it supports *only* TLS you can drop the ifdef too

Reply:
Yes, we are TLS only.